<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Button_dialog</class>
 <widget class="QDialog" name="Button_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 34);</string>
  </property>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down_pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>61</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images/aqua_image.qrc">
     <normaloff>:/images/down.webp</normaloff>:/images/down.webp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>65</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="right_pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images/aqua_image.qrc">
     <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="left_pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images/aqua_image.qrc">
     <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>65</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>61</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../images/aqua_image.qrc">
     <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>55</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images/aqua_image.qrc"/>
 </resources>
 <connections/>
</ui>
